package.json

```diff
@@ -1,6 +1,17 @@
 {
-  "name": "contributing-docs",
+  "name": "@bitwarden/contributing-docs",
   "version": "0.0.0",
+  "description": "Bitwarden Contributing Docs",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bitwarden/contributing-docs.git"
+  },
+  "author": "Bitwarden Inc. <hello@bitwarden.com> (https://bitwarden.com)",
+  "license": "GPL-3.0",
```

Comment:
The code is licensed GPL, the docs are CC BY-NC-ND

Reply:
Hah, as soon as I pushed my updates and my above comment this refreshed so I saw your proposal -- GitHub comment edits don't get emailed I guess. At any rate we arrived at the same conclusion! 🎉